SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -549,6 +555,32 @@ FunctionDef {
 		^().putPairs(this.keyValuePairsFromArgs)
 	}
 
+	makeNonOverlappingKeywordArgsEvent {|argsArray, keywordArgsEvent, omitArgNameIfFirst|
```

Comment:
This method is a little opaque still. I'll see if I can give you some suggestions later.

Reply:
I've renamed these in the latest to `makePerformableEnvir` and `makePerformableArray`.